lib/bamboo/test.ex

```diff
@@ -65,8 +67,44 @@ defmodule Bamboo.Test do
   @doc """
   Imports Bamboo.Test and Bamboo.Formatter.format_email_address/2
   """
+  defmacro __using__(process_name: process_name) do
+    quote do
+      setup tags do
+        if tags[:async] do
```

Comment:
I could not figure out how to test this, since it would raise before the tests ever ran (which is the point :P) So I'm not really sure how to test it. Might not be worth it?